homeassistant/components/asuswrt.py

```diff
@@ -37,7 +39,9 @@
         vol.Optional(CONF_REQUIRE_IP, default=True): cv.boolean,
         vol.Exclusive(CONF_PASSWORD, SECRET_GROUP): cv.string,
         vol.Exclusive(CONF_SSH_KEY, SECRET_GROUP): cv.isfile,
-        vol.Exclusive(CONF_PUB_KEY, SECRET_GROUP): cv.isfile
+        vol.Exclusive(CONF_PUB_KEY, SECRET_GROUP): cv.isfile,
+        vol.Optional(CONF_SENSORS, default=SENSOR_TYPES): vol.All(
```

Comment:
Does it work set this key with no value to configure no sensors? I think it should.

Reply:
Yes, but maybe it should be the other way around?

Reply:
What's the most common use case, all sensors or no sensors?

Reply:
I would say no sensors, I'll change it